### DISCUSSION TITLE
SQLitebrowser for sqlcipher database?

### DISCUSSION THREAD

**Discussion by dbyy** [sqlitebrowser/sqlitebrowser#3418] [Ideas]
I will post this as a question and/or as a suggestion for improvement..
Without going deep into the technical details.. would it be possible to create a DB Browser for sqlcipher (the encrypted version of SQLite)?
KMyMoney is using sqlcipher as database backend.
What are your thoughts about this?

**Comment by justinclift** [comment]
We already do have a SQLCipher version of the GUI. :smile:

If you're on Windows or macOS, it's part of the existing downloads.  If you're using Linux it can be a bit trickier as you need to grab the right thing.

Which Operating System are you using? :smile:

**Reply by dbyy** [reply]
I'm on Linux Mint v20/Ubuntu 20.04

**Reply by justinclift** [reply]
Cool.  In that case, the easiest option will be for you to grab our latest .AppImage development build:

https://github.com/sqlitebrowser/sqlitebrowser/releases/tag/continuous

That gets rebuilt automatically when every new commit to our repo.  It should be fairly stable at the moment. :smile:
